test/factories/inventory_upload_factories.rb

```diff
@@ -23,50 +23,50 @@
 
 FactoryBot.define do
   factory :katello_content_view, :class => Katello::ContentView do
-    sequence(:name) { |n| "Database#{n}" }
-    description {"This content view is for database content"}
+    sequence(:name) { |n| "Database#{n} " }
+    description { "This content view is for database content" }
     association :organization, :factory => :katello_organization
 
     trait :composite do
-      composite {true}
+      composite { true }
     end
   end
 end
 
 FactoryBot.define do
   factory :katello_k_t_environment, :aliases => [:katello_environment], :class => Katello::KTEnvironment do
-    sequence(:name) { |n| "Environment#{n}" }
-    sequence(:label) { |n| "environment#{n}" }
+    sequence(:name) { |n| "Environment#{n} " }
```

Comment:
This doesn't make sense, why the whitespace inside the string?

Reply:
oops, good catch !